components/mypage/DoneItemDetail.tsx

```
@@ -4,210 +4,150 @@ import React from "react";
+            <span className="font-SUIT_Variable font-medium text-xs leading-[1.2] tracking-[-0.02em] text-Color-gray-60">
+              {todo.date}
+            </span>
+          </div>
+        )}
+      </div>
+
+      {todo.todoResult?.fileUrl && (
+        <div className="mt-2">
+          {/* {todo.attachment.type === "image" ? ( TODO: how to distinguish image? */}
+          {false ? (
+            <div className="w-[116px] h-[116px] border border-Color-gray-30 rounded-lg overflow-hidden">
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)